tests/benchmark/benchmark_between_releases/benchmark_between_releases.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Build the benchmark image for release 1 from Dockerfile
+echo "Building image for release 1"
+export FULL_BASE_IMAGE="${DOCKER_NAMESPACE}/python:${TAG1}"
+export TAG="${TAG1}"
+envsubst <"Dockerfile".in >"Dockerfile" '$FULL_BASE_IMAGE $TAG'
+docker build -t benchmark_1 .
+rm Dockerfile
+
+# Build the benchmark image for release 2 from Dockerfile
+echo "Building image for release 2"
+export FULL_BASE_IMAGE="${DOCKER_NAMESPACE}/python:${TAG2}"
+export TAG="${TAG2}"
+envsubst <"Dockerfile".in >"Dockerfile" '$FULL_BASE_IMAGE $TAG'
+docker build -t benchmark_2 .
+rm Dockerfile
+
+echo "Successfully built images"
+
+# Start running the containers
+docker run -it --name benchmark_1 -h CONTAINER1 -v /"${TAG1}" benchmark_1 ls
```

Comment:
Instead of using a data volume + docker run ls + docker cp, it might work to directly mount a host directory and then copy to it with something like (`/export` is arbitrary, it could be some other name):
    
    mkdir release1/
    docker run --name benchmark_1 --volume="${PWD}"/release1:/export /bin/bash -c "cp /app/py*.json /export/"